### PR TITLE
fix(security) cp2foss SQL injection vulnerability

### PR DESCRIPTION
**Pull request by timurphy**
Two SQL queries in the cp2foss utility insert variables into the SQL string, which creates an SQL injection vulnerability. This PR converts those queries into prepared statements using DbManager. SELECT * has also been replaced with SELECT <fields> for efficiency.

### CODE REVIEW

File: src/cli/cp2foss.php [hunk 1]
```diff
@@ -175,19 +175,16 @@ function GetFolder($FolderPath, $Parent = NULL) {
   }
   /* See if it exists */
   $SQLFolder = str_replace("'", "''", $folderHead);
```
Comment (maximilianhuber) [inline src/cli/cp2foss.php]: Using prepared statements, you do not longer have to have this escaping here

Comment (timurphy) [inline src/cli/cp2foss.php]: Thanks. I'll run more tests for this case before committing the change.

**Comment by maximilianhuber** [comment]
Code looks good to me, I will test the functionallity soon

**Comment by maximilianhuber** [comment]
I have found some problems but I am not yet shure, whether they are related to your PR. I will do some further checks.

**Comment by timurphy** [comment]
Removed the `str_replace` and ran some extra tests containing `'` and `"` characters.

**Pull request merged**